### PR TITLE
Fix links in docs.

### PR DESCRIPTION
**Pull request by eskimor**

### CODE REVIEW

File: roadmap/implementers-guide/src/node/availability/availability-distribution.md [hunk 1]
```diff
@@ -46,12 +46,9 @@ backers, not all of them are required to have the needed chunk. This in turn
 could lead to low throughput, as we have to wait for a fetches to fail,
 before reaching a validator finally having our chunk. We do rank back validators
 not delivering our chunk, but as backers could vary from block to block on a
-perfectly legitimate basis, this is still not ideal. See the following issues
+perfectly legitimate basis, this is still not ideal. See issues [2509](https://github.com/paritytech/polkadot/issues/2509) and [2512](https://github.com/paritytech/polkadot/issues/2512)
 for more information:
```
Comment (bkchr) [inline roadmap/implementers-guide/src/node/availability/availability-distribution.md:50]:
```suggestion
for more information.
```

**Review by bkchr — APPROVED** [review]

File: roadmap/implementers-guide/src/node/availability/availability-distribution.md (hunk 1 above)
Comment (eskimor) [reply roadmap/implementers-guide/src/node/availability/availability-distribution.md:50]: Very confused. I could have sworn I changed that already. Anyway, thanks!

**Comment by ordian** [comment]
bot merge

**Comment by ordian** [comment]
CI failure should be fixed in https://github.com/paritytech/polkadot/pull/2557.

**Comment by ordian** [comment]
bot merge

**Pull request merged**